lib/spack/docs/configuration.rst

```diff
@@ -132,36 +210,6 @@ The ``buildable`` does not need to be paired with external packages.
 It could also be used alone to forbid packages that may be
 buggy or otherwise undesirable.
 
-.. _system-packages:
-
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-Extracting System Packages
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-In some cases, using false paths for system packages will not work.
-Some builds need to run binaries out of their dependencies, not just
-access their libraries: the build needs to know the real location of
-the system package.
-
-In this case, one can create a Spack-like single-package tree by
-creating symlinks to the files related to just that package.
-Depending on the OS, it is possible to obtain a list of the files in a
-single OS-installed package.  For example, on RedHat / Fedora:
-
-.. code-block:: console
-
-   $ repoquery --list openssl-devel
-   ...
-   /usr/lib/libcrypto.so
-   /usr/lib/libssl.so
-   /usr/lib/pkgconfig/libcrypto.pc
-   /usr/lib/pkgconfig/libssl.pc
-   /usr/lib/pkgconfig/openssl.pc
-   ...
-
-Spack currently does not provide an automated way to create a symlink
-tree to these files.
-
```

Comment:
This section needs to be modified, not removed.  The extraction is no longer needed for system packages.  But it would be needed for extracting random packages out of non-system trees.  For example, using one package out of a MacPorts tree.  Or something out of some random tree lying around.